sensors/light.py

```diff
@@ -1,24 +1,29 @@
 import time
 import grovepi
+import requests
 import services.telemetry as telemetry
-import services.config as config
 
-sensor = config.getValue("sensor_light")
-sleepTime = config.getValue("sleep_in_seconds_light")
-grovepi.pinMode(sensor,"INPUT")
+def monitorLight(currentConfig):
```

Comment:
Wird die hier noch irgendwo verwendet? 
Wenn nicht würde ich ne ``__main__`` rausmachen, da mit ``currentConfig = config.getConfig()`` die Config auslesen und dann in ner while schleife mit sleep ``sendLight(currentConfig)`` ausführen (so wies jetzt auch schon ist).
Dann könnte man die Skripte auch autark nutzen, statt sich immer eine neue config zu generieren.

Reply:
Dann bekommst du wieder das gleiche Problem mit den Imports, wenn du das nicht aus dem Projekt-root startest 😕

Reply:
Wir könnten das benutzen, wenn wir start.py Paramter übergeben um einen bestimmten Sensor abzufragen.

Reply:
Prinzipiell wäre es natürlich nett alle Skripte einzeln starten zu können, ich finde allerdings die Möglichkeit wie es aktuell ist (einfach keinen Wert in der `config.json` eintragen) noch ein Stück intuitiver. So kann man einfach sagen "Starte mit dem Monitoring" und alle Sensoren werden abgegriffen, die auch konfiguriert sind.

Reply:
Die `config.json` wird ja im Idealfall nur einmalig generiert, und dann bei Bedarf direkt abgeändert.

Reply:
> 
> 
> Dann bekommst du wieder das gleiche Problem mit den Imports, wenn du das nicht aus dem Projekt-root startest 😕

dann fix das gefälligst 👿 😆

Reply:
> 
> 
> Wir könnten das benutzen, wenn wir start.py Paramter übergeben um einen bestimmten Sensor abzufragen.

hört sich gut an